packages/app/src/reducers/index.js

```diff
@@ -43,12 +43,9 @@ export {
     fromChart,
 }
 
-export const sGetAxisSetupItems = state =>
-    fromUi.sGetAxisSetup(state).map(obj => {
-        const metadata = fromMetadata.sGetMetadata(state)[obj.id]
-
-        return {
-            ...obj,
-            name: metadata ? metadata.name : '',
-        }
-    })
+export const sGetSeriesSetupItems = state =>
```

Comment:
Maybe just `SeriesItems`, not `SeriesSetupItems`

Reply:
I used this name to avoid confusion with `series`, which is a prop in `options` fetched by the API (used a lot in this PR). Compared to the `SeriesSetupItems` which instead is based on the `sGetAxisSetup` in the ui reducer ([here](https://github.com/dhis2/data-visualizer-app/blob/feat/DHIS2-9054-series-options/packages/app/src/reducers/ui.js#L307)). But I'm happy to change it if you insist :v:

Reply:
Ok no, then that's okay :+1: